### ISSUE TITLE
NR CLI Translate base64 encoded urls command

### ISSUE THREAD

**Issue by jpvajda** [newrelic/newrelic-cli#623]
In the Seldon era, the NR URLs produced were human readable. With NR1 all of the urls are base64 encoded JSON strings, these are not human readable. There has been advancements to make them more human readable, but there are still some use cases where it is helpful to know the Application ID or Condition ID, information that is not readily available from the UI.

 it takes an existing NR1 URL, and parses it from base64 to a human readable JSON format. It even handles the situations where the encoded string contains an encoded entity ID, as it is quite often this entity ID information that we require.

### TO DO
Add a command that implements the behavior of the [internal tool](https://source.datanerd.us/nfoley/base64decoder) that was built to take care of this use case and add the  functionality to the NR CLI.

### Command 

`translate`  

-h, --help  
 --format string   output text format [JSON, Text, YAML] (default "JSON")
 --plain           output compact text

Example URL (works for any NR1 URL)

```
    https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJhcG0tbmVyZGxldHMub3ZlcnZpZXciLCJpc092ZXJ2aWV3Ijp0cnVlLCJlbnRpdHlJZCI6Ik1UWXdOamcyTW54QlVFMThRVkJRVEVsRFFWUkpUMDU4TlRBeE1UWXpORGcifQ==&sidebars[0]=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5hY3Rpb25zIiwiZW50aXR5SWQiOiJNVFl3TmpnMk1ueEJVRTE4UVZCUVRFbERRVlJKVDA1OE5UQXhNVFl6TkRnIiwic2VsZWN0ZWROZXJkbGV0Ijp7Im5lcmRsZXRJZCI6ImFwbS1uZXJkbGV0cy5vdmVydmlldyIsImlzT3ZlcnZpZXciOnRydWV9fQ==&platform[timeRange][duration]=1800000&platform[$isFallbackTimeRange]=true

```

current Output in internal tool

```

    ------------------------
    485171|APM|APPLICATION|676227677
    ------------------------
    {"nerdletId":"apm-nerdlets.overview","isOverview":true,"entityId":"NDg1MTcxfEFQTXxBUFBMSUNBVElPTnw2NzYyMjc2Nzc"}

```

desired Output 

```
{"nerdletId":"apm-nerdlets.overview","isOverview":true,"entityId":"NDg1MTcxfEFQTXxBUFBMSUNBVElPTnw2NzYyMjc2Nzc", "domain": "APM", "type": "APPLICATION", "accountId": 485171, "applicationId": 676227677}
```

**Comment by jpvajda** [comment]
https://github.com/GlenOFoghl was interested in contributing this feature.

**Comment by sanderblue** [comment]
Resolved in #656
